.teamcity/Gradle_Check/model/CIBuildModel.kt

```diff
@@ -404,6 +405,7 @@ enum class TestType(val unitTests: Boolean = true, val functionalTests: Boolean
     allVersionsCrossVersion(false, true, true, 240),
     parallel(false, true, false),
     noDaemon(false, true, false, 240),
+    instant(false, true, false),
```

Comment:
I think you also need to add a `buildType` with name `instantTest` for the test type to `build.gradle.kts`: https://github.com/gradle/gradle/blob/e5f8a419351fd178ccb1fcfe898e1650f2d66871/build.gradle.kts#L53-L79

I suppose the name of the task will be `instantIntegTest`, I hope that will be fine for you. Maybe the `buildType` will also need to be added on `release`, given that we use a single TC pipeline for both `release` and `master`. Though let's try it out first.